File: Day1/Cricket.py
class Cricket:
    score = 0
    def __init__(self,val):
        self.val = val
    def incr(self):
        Cricket.score = Cricket.score + self.val

fb = Cricket(56)
sb = Cricket(33)
ext = Cricket(11)
fb.incr()
sb.incr()
ext.incr()
print(Cricket.score)

File: JavaCore/src/com/zorin/test/Cricket.java
package com.zorin.test;

public class Cricket {

	static int score;
	
	public void incr(int x) {
		score = score+x;
	}
	
	public static void main(String[] args) {
		Cricket fb=new Cricket();
		Cricket sb=new Cricket();
		Cricket ext=new Cricket();
		
		fb.incr(67);
		sb.incr(11);
		ext.incr(7);
		
		System.out.println("Total Score is  " +Cricket.score);
	}
}
